sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -0,0 +1,120 @@
+projectName: 'elife-data-pipeline'
+importedTimestampFieldName: 'data_hub_imported_timestamp'
+webApi:
+  #observer api
+  - dataset: '{ENV}'  for observer
+    table: test_published_research_article
+    datataUrl:
+      urlExcludingConfigurableParameters: https://observer.elifesciences.org/report/published-research-article-index?format=json
+  #elifescience people api
+  - dataset: '{ENV}'
+    table: test_people_api
+    datataUrl:
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
+      configurableParameters:
+        pageSize: per-page
```

Comment:
it might be good to indicate that a parameter is a parameter name. Most of them seem to be, except for the default values. Maybe `pageSizeParameterName` (I know it's a bit longer)